docs/guides/nrfconnect_android_commissioning.md

```diff
@@ -60,27 +61,33 @@ applications:
 You need the following hardware and software for commissioning the nRF Connect
 accessory using Android CHIPTool:
 
--   Two nRF52840 DK (PCA10056)
-
-    -   One nRF52840 DK is needed for running the
-        [OpenThread Radio Co-Processor](https://openthread.io/platforms/co-processor)
-        firmware. You can replace this DK with another compatible device, such
-        as the nRF52840 Dongle.
-    -   One nRF52840 DK is needed for running the example application. You can
-        replace this DK with another compatible device, such as the nRF5340 DK.
-
--   Smartphone compatible with Android 8.0 or later
--   Raspberry Pi Model 3B+ or newer (along with an SD card with at least 8 GB of
-    memory)
--   Wi-Fi Access Point supporting IPv6 (without the IPv6 Router Advertisement
+-   1x smartphone with Android 8+
+-   1x Wi-Fi Access Point supporting IPv6 (without the IPv6 Router Advertisement
     Guard enabled on the router)
+-   1x nRF52840 DK (PCA10056) for running the example application. You can
```

Comment:
Can we clarify here which DKs are compatible with Thread and Wi-Fi?

Reply:
Added clarification.